lease/lessor.go

```diff
@@ -192,6 +241,22 @@ func (le *lessor) Recover(b backend.Backend, dr DeleteableRange) {
 	le.initAndRecover()
 }
 
+func (le *lessor) runLoop() {
```

Comment:
@heyitsanthony Add the expiration management loop to make this commit more obvious.